examples/ultralytics-yolo/benchmark.py

```diff
@@ -271,6 +277,15 @@ def parse_args(arguments=None):
             "post-processing. Defaults to use standard YOLOv3/YOLOv5 anchors"
         ),
     )
+    parser.add_argument(
```

Comment:
the recipe, if any was used, is already present in the state dict exported from ultralytics. Check the loading logic that we have in the models/export.py class for yolov5. We shouldn't need to have the user supply a recipe here